packages/@react-spectrum/s2/src/TreeView.tsx

```diff
@@ -110,13 +110,13 @@ function TreeView(props: TreeViewProps, ref: DOMRef<HTMLDivElement>) {
       layout={ListLayout}
       layoutOptions={{
         rowHeight: scale === 'large' ? 50 : 40,
-        gap: isDetached ? 4 : 0
+        gap: isDetached ? 2 : 0
```

Comment:
Are they supposed to be that close together? Figma has them much further apart (looks like 8px):

<img width="290" alt="image" src="https://github.com/user-attachments/assets/9e128836-39ef-468e-a194-2cae93be525d" />

Reply:
Not in the tokens file, it says 2px, and I checked against the token as well in the latest release. `tree-view-item-to-item-detached`

Reply:
sounds like we need to ask design then

Reply:
started conversation